packages/back-end/src/models/GeneratedHypothesis.ts

```diff
@@ -0,0 +1,228 @@
+import { omit } from "lodash";
+import mongoose from "mongoose";
+import uniqid from "uniqid";
+import { ReqContext } from "@back-end/types/organization";
+import { GeneratedHypothesisInterface } from "@back-end/types/generated-hypothesis";
+import { ExperimentInterface } from "@back-end/types/experiment";
+import { createExperiment } from "./ExperimentModel";
+import { upsertWatch } from "./WatchModel";
+import { createVisualChangeset } from "./VisualChangesetModel";
+import { createFeature } from "./FeatureModel";
+
+type GeneratedHypothesisDocument = mongoose.Document &
+  GeneratedHypothesisInterface;
+
+const generatedHypothesisSchema = new mongoose.Schema({
+  id: String,
+  uuid: String,
+  createdAt: Date,
+  organization: String,
+  url: String,
+  hypothesis: String,
+  experiment: {
+    type: String,
+    index: true,
+    require: false,
+  },
+});
```

Comment:
We should be using the BaseModel for all new DB models. It enforces best practices that are missing from here. For example, this code has a security vulnerability. You can fetch by uuid and it doesn't verify that the organization field matches your request context. With BaseModel, this would be handled automatically.